core/tests/protractor/historyTab.js

```diff
@@ -97,70 +97,85 @@ describe('Exploration history', function() {
         highlighted: true
       },
       10: {
-        text: '      feedback: []',
+        text: '      feedback:',
         highlighted: true
       },
       11: {
+        text: '        audio_translations: {}',
+        highlighted: false
```

Comment:
Change this to `highlighted: true`. The boolean value shows whether the line is highlighted in the diff view.